helpers.js

```diff
@@ -42,9 +33,37 @@ function logToActivityLog(message) {
   console.error(message);
 }
 
+async function validatePaths(paths) {
+  const pathsArray = Array.isArray(paths) ? paths : [paths];
+
+  const pathPromises = pathsArray.map(pathExists);
```

Comment:
If we take a `path` and `check` if `pathExists` then the result is `pathExistanceCheck`.
If it's not synchronous then it's also a `promise`. (:heavy_check_mark:)
If it's plural then the plural form should be used. (:heavy_check_mark:)

These are details, and barelly make any difference here, but let's consider this as an excercise to master better naming.